lib/wovnrb.rb

```diff
@@ -47,27 +47,31 @@ def call(env)
       status, res_headers, body = @app.call(headers.request_out)
 
       if res_headers["Content-Type"] =~ /html/
-        unless @store.settings['ignore_globs'].any?{|g| g.match?(headers.pathname)}
-          # If the user defines a token for this request, use it instead of the config token
-          request = Rack::Request.new(env)
-          if @store.valid_token?(request.params['wovn_token'])
-            @store.settings['project_token'] = request.params['wovn_token']
+
+        request = Rack::Request.new(env)
+        unless request.params['wovn_disable'] == true
```

Comment:
Looks too much.
Why don't you just `unless request.params['wovn_disable']`?

Reply:
I wanted to be sure it’s true and the user didn’t put something like “OK” that will also return a true value

I want to check type and truthiness